app/views/collection/show.html.slim

```diff
@@ -160,8 +160,8 @@
       =link_to t('.show_more'), collection_notes_path(collection_id: @collection.slug), class: 'button outline round'
 
     h3 =t('.recent_edits')
-    -cache @collection.most_recent_deed_created_at do
-      =deeds_for({ :collection => @collection, :limit => 20, :types => DeedType.collection_edits, :long_view => true })
+    -cache [@collection.id, @collection.most_recent_deed_created_at, :collection_show] do
```

Comment:
I decided we should also add `:collection_show` as key to indicate the page this is being used. We can see in watchlist that it will cause another collision if we don't specify that this is for `:collection_show`